### DISCUSSION TITLE
Applied for GitHub Student Pack

### DISCUSSION THREAD

**Discussion by yigiteren** [community/community#22823] [New to GitHub]
<p>Hi, I’ve applied for GitHub Student Pack a couple of hours ago. How long may it take to get approved? I’ve sent my students ID and applied with my student e-mail.</p>

**Comment by nyahbhinghiPrincess** [comment, marked as answer]
<p>Hi <a class="mention" href="/yigiteren">@yigiteren</a>,</p>
<p>Unfortunately, it can take a little bit of time for the application review to be completed. We do our best to respond as quickly as possible to all tickets and are currently working on some efforts to allow us to shorten our average response time.</p>
<p>Thanks for your patience and understanding!</p>

**Comment by christinaet** [comment]
<p>I have been waiting for the approval for two weeks. Due to Corona Virus all the schools in New York is closed and we are switching to virtual learning. I really want to keep up with my classes and do school work. Can you please advise on the timeline in reviewing the application? </p>

**Comment by AndreaGriffiths11** [comment]
<p>Sorry for the delay and thank you for your patience. The number of applications we receive has increased :chart_with_upwards_trend: thanks to the incredible companies who have joined the Pack. They’re taking us some time to review and driving up support times as well. We promise it’s worth the wait.</p>

**Comment by anuragnode** [comment]
<p>Due to covid i can’t attend my classes and its taking too much time in approval</p>
